test-requirements.txt

```
@@ -0,0 +1 @@
+odoo16-addon-commission@git+https://github.com/OCA/commission/pull/561/head#subdirectory=setup/commission
```

I test non si stanno eseguendo per questo errore
> fatal: repository 'https://github.com/OCA/commission/pull/561/head/' not found

(da https://github.com/OCA/l10n-italy/actions/runs/12182965960/job/33983246738?pr=4403#step:4:192)

probabile che sia perché questo è da correggere:
```suggestion
odoo-addon-commission @ git+https://github.com/OCA/commission.git@refs/pull/561/head#subdirectory=setup/commission
```
(vedi https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s))

Qualche test fallirà comunque per https://github.com/OCA/l10n-italy/issues/4461, ma almeno il resto dei test verranno eseguiti.